docusaurus/docs/tutorials/ui-extensions/exposed-components.md

```diff
@@ -0,0 +1,77 @@
+---
+id: exposed-components
+title: Exposed components
+description: Share functionality between plugins with exposed components
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - extensions
+  - ui-extensions
+sidebar_position: 30
+---
+
+Exposed components are a way to share functionality between plugins.
+
+## Exposing a component from an app plugin
+
+```tsx
+import pluginJson from './plugin.json';
+
+export const plugin = new AppPlugin()
+  // You can also expose multiple components from the same app plugin
+  .exposeComponent({
+    // Important!
+    // The `id` should always be prefixed with your plugin id, otherwise it won't be exposed.
+    id: `${pluginJson.id}/reusable-component`,
+    title: 'Reusable component',
+    description: 'A component that can be reused by other app plugins.',
+    component: ({ name }: { name: string }) => (
+      <div>Hello {name}!</div>
+    )
+  })
+```
+
+## Using an exposed component
```

Comment:
Move to new page

Title: Use an exposed component

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/954/commits/999f5a2263de23d9f7fd41e5f47d07843fc71ed7.